### ISSUE TITLE
How to save output Labels in csv file.

### ISSUE THREAD

**Issue by hammad2008** [DeMoriarty/fast_pytorch_kmeans#13]
Hello @DeMoriarty  I have implement your code. I want to save the labels/Index in CSV format How can i do that?
I have applied for Loop for 15 times to get 15 clusters. and save each index values in separate column eg: first column shows when cluster=1, then 2nd column shows when cluster=2

```
from fast_pytorch_kmeans import KMeans
import torch
from time import time
import numpy as np
import matplotlib.pyplot as plt
array = np.loadtxt('Dataset1.csv', delimiter=',')
twofeatures = array[:, [1, 3]]
#print(twofeatures)
torchfeatures = torch.from_numpy(twofeatures)
gpufeature = torchfeatures.cuda()
gpufeature = gpufeature.to(torch.float)
lenebl = []
gpu_times = []
kmeans_times=[]
for i in range(1,15):
    start_kmeans = time()
    kmeans = KMeans(n_clusters=i, mode='euclidean', verbose=1)
    labels = kmeans.fit_predict(gpufeature)
    kmeans_time = time() - start_kmeans 
    kmeans_times.append(kmeans_ti
    print(f'Kmeans time: {kmeans_time}')
    index = labels.cpu().data.numpy()
    
```
[Dataset1.csv](https://github.com/DeMoriarty/fast_pytorch_kmeans/files/11085359/Dataset1.csv)
